client/extensions/woocommerce/components/woocommerce-logo/index.js

```diff
@@ -0,0 +1,62 @@
+						id="Shape"
+						fill="#FFFFFF"
+						fillRule="nonzero"
+					/>
+					<path
+						d="M270.9,35.7 C264.2,42.3 260.9,50.7 260.9,60.9 C260.9,71.8 264.2,80.7 270.8,87.4 C277.4,94.1 286,97.5 296.7,97.5 C299.8,97.5 303.3,97 307.1,95.9 L307.1,79.7 C303.6,80.7 300.6,81.2 298,81.2 C292.7,81.2 288.5,79.4 285.3,75.9 C282.1,72.3 280.5,67.5 280.5,61.4 C280.5,55.7 282.1,51 285.2,47.4 C288.4,43.7 292.3,41.9 297.1,41.9 C300.2,41.9 303.5,42.4 307.1,43.4 L307.1,27.2 C303.8,26.3 300.1,25.9 296.2,25.9 C286,25.8 277.6,29.1 270.9,35.7 Z M340.3,25.8 C331.1,25.8 323.9,28.9 318.7,35 C313.5,41.1 311,49.7 311,60.7 C311,72.6 313.6,81.7 318.7,88 C323.8,94.3 331.3,97.5 341.1,97.5 C350.6,97.5 357.9,94.3 363,88 C368.1,81.7 370.7,72.8 370.7,61.4 C370.7,50 368.1,41.2 362.9,35 C357.6,28.9 350.1,25.8 340.3,25.8 Z M348.2,77.8 C346.4,80.6 343.7,82 340.3,82 C337.1,82 334.7,80.6 333,77.8 C331.3,75 330.5,69.4 330.5,60.9 C330.5,47.8 333.8,41.3 340.5,41.3 C347.5,41.3 351.1,47.9 351.1,61.2 C351,69.4 350,75 348.2,77.8 Z M420.1,27.7 L416.5,43 C415.6,46.9 414.7,50.9 413.9,55 L411.9,65.6 C410,55 407.4,42.4 404.1,27.7 L380.9,27.7 L372.2,95.8 L389.6,95.8 L394.3,48.9 L406.2,95.8 L418.6,95.8 L430,49 L434.9,95.8 L453.1,95.8 L443.9,27.7 L420.1,27.7 Z M503.4,27.7 L499.8,43 C498.9,46.9 498,50.9 497.2,55 L495.2,65.6 C493.3,55 490.7,42.4 487.4,27.7 L464.2,27.7 L455.5,95.8 L472.9,95.8 L477.6,48.9 L489.5,95.8 L501.9,95.8 L513.2,49 L518.1,95.8 L536.3,95.8 L527.1,27.7 L503.4,27.7 Z M560,68.9 L576.3,68.9 L576.3,54.8 L560,54.8 L560,42.3 L578.8,42.3 L578.8,27.8 L541.6,27.8 L541.6,95.9 L578.9,95.9 L578.9,81.4 L560,81.4 L560,68.9 Z M630.7,58.1 C632.6,55 633.6,51.8 633.6,48.5 C633.6,42.1 631.1,37 626.1,33.3 C621.1,29.6 614.2,27.7 605.6,27.7 L584.2,27.7 L584.2,95.8 L602.6,95.8 L602.6,64.8 L602.9,64.8 L617.8,95.8 L637.2,95.8 L622.5,65.1 C626,63.5 628.8,61.2 630.7,58.1 Z M602.5,57 L602.5,40.8 C606.9,40.9 610,41.6 611.9,43 C613.8,44.4 614.7,46.6 614.7,49.8 C614.7,54.5 610.6,56.9 602.5,57 Z M644.4,35.7 C637.7,42.3 634.4,50.7 634.4,60.9 C634.4,71.8 637.7,80.7 644.3,87.4 C650.9,94.1 659.5,97.5 670.2,97.5 C673.3,97.5 676.8,97 680.6,95.9 L680.6,79.7 C677.1,80.7 674.1,81.2 671.5,81.2 C666.2,81.2 662,79.4 658.8,75.9 C655.6,72.3 654,67.5 654,61.4 C654,55.7 655.6,51 658.7,47.4 C661.9,43.7 665.8,41.9 670.6,41.9 C673.7,41.9 677,42.4 680.6,43.4 L680.6,27.2 C677.3,26.3 673.6,25.9 669.7,25.9 C659.6,25.8 651.1,29.1 644.4,35.7 Z M704.1,81.2 L704.1,68.8 L720.4,68.8 L720.4,54.7 L704.1,54.7 L704.1,42.2 L723,42.2 L723,27.7 L685.8,27.7 L685.8,95.8 L723.1,95.8 L723.1,81.3 L704.1,81.3 L704.1,81.2 Z"
+						id="Shape"
+						fill="#527994"
+						fillRule="nonzero"
+					/>
+					{ /* eslint-enable max-len */ }
+				</g>
+			</g>
+		</svg>
+	);
+};
+
+WooCommerceLogo.propTypes = {
```

Comment:
Only thing I could see changing here is to have some default propTypes so these aren't required.